NetLicensingClient/RestController/NetLicensingAPI.cs

```diff
@@ -98,6 +101,7 @@ public static netlicensing request(Context context, Method method, String path,
             request.PreAuthenticate = true;
             request.Accept = "application/xml";
             request.SendChunked = false;
+            request.Timeout = timeoutInSeconds * 1000;
```

Comment:
To avoid calc here, attribute can specify timeoutInMsec already

Reply:
That could also be an option. I just think milliseconds is too high granularity. But yes, it could be timeout in milliseconds :)